packages/templates/usa-documentation/includes/_usa-docs-inner.twig

```diff
@@ -1,55 +1,53 @@
+{% set settings = {
```

Comment:
Renaming to `sidenav_settings` would make it immediately clear that it's a settings object for sidenav.

```suggestion
{% set sidenav_settings = {
```

Reply:
I can dig it!